### ISSUE TITLE
Would it be possible to lower friendsofphp/php-cs-fixer version requirement to v3.4 for drupal/core-recommended:^9.x to work with it (on a specific version, eg: 0.3.9)

### ISSUE THREAD

**Issue by eedev** [elyby/php-code-style#15]
Hello there 👋 

I'm sorry to trouble you again! 

I need to use your project with Drupal 9 but, the friendsofphp/php-cs-fixer:^3.11 requirement that was introduced a few days before your 0.4.0 release is not compatible with Drupal 9. 

This is because drupal/core-recommended:^9.0 (until ^10.0) depends on `symfony/console: ~v4.4.49`, while `friendsofphp/php-cs-fixer:^3.11` requires `symfony/console: ^5.4 || ^6.0`.

Although [symfony/console](https://packagist.org/packages/symfony/console):`^4.4.20 || ^5.1.3 || ^6.0` was suitable for https://packagist.org/packages/friendsofphp/php-cs-fixer#v3.4.0, it is no longer compatible with https://packagist.org/packages/friendsofphp/php-cs-fixer#v3.5.0.

Would it be feasible go back in time and to create a tag, maybe something like 0.3.9 using friendsofphp/php-cs-fixer:^3.4 required ?

Thank you for your assistance 😄 .

**Comment by erickskrauch** [comment]
Do you want to use our entire codestyle or just custom rules?

**Comment by eedev** [comment]
This is what I use : 

```yaml
parameters:
  rules:
    Drupal/blank_line_before_end_of_class: false
    Ely/blank_line_around_class_body: true
    Ely/blank_line_before_return: true
    Ely/line_break_after_statements: true
```

**Comment by erickskrauch** [comment]
I sat down to do it, but then I realized the problem.

The `0.3` release [doesn't support PHP-CS-Fixer 3](https://github.com/elyby/php-code-style/blob/0.3.0/composer.json#L20). And the `0.4` release came out with a set of rules which require at least PHP-CS-Fixer 3.13. If I create a tag from the `0.4` release with the lowered PHP-CS-Fixer requirement, then I will create an unstable release because the ruleset will not work because it relies on newer fixers which are only available in higher versions of PHP-CS-Fixer.

All these contradictions made me think that I don't want to do this at all. [Drupal 10 has already been released](https://www.drupal.org/project/drupal/releases/10.0.0), so the question is only to do an upgrade.

As a temporary solution you can make a fork of my repository and downgrade to PHP-CS-Fixer in it (you are not responsible for the ruleset) and use it in your project:

1) Fork.
2) Edit `composer.json` file in a way that matches your wanted PHP-CS-Fixer version.
3) Add this to the `compose.json` of your project:
   ```json
   {
       "require-dev": {
           "ely/php-code-style": "dev-yourbranch"
       },
       "repositories": [
           {
               "type": "github",
               "url": "https://github.com/eedev/php-code-style.git"
           }
       ]
   }
   ```
4) Execute in terminal `composer update ely/php-code-style`.

**Comment by eedev** [comment]
Hello 👋 

Thank you for taking the time!

IMO code sniffing is especially useful on these older sites that we have to maintain, but I understand it's something complicated to do. 
When I tried, I started from a specific commit you made a few days before tagging 0.4.0, not 0.3.0 directly.

I've found another solution that is aliasing `"[friendsofphp/php-cs-fixer](https://packagist.org/packages/friendsofphp/php-cs-fixer)": "[3.4 as 3.13](https://packagist.org/packages/friendsofphp/php-cs-fixer)".

Since I am building a QA package set (yours is required in it), my goal was to be able to only require --dev my package and be done with it. Only one line in the composer.json 😁.

Again thanks for taking the time, have a good day! 😃
